modules/apps/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/util/test/CalendarUtilTest.java

```diff
@@ -272,6 +273,14 @@ public void testToCalendarJSONObject() throws Exception {
 			CalendarResourceTestUtil.addCalendarResource(
 				_groupLocalService.getGroup(TestPropsValues.getGroupId()));
 
+		_calendarResourceService.updateCalendarResource(
+			calendarResource.getCalendarResourceId(),
+			HashMapBuilder.put(
+				LocaleUtil.getDefault(),
+				"lp'\"></option><img onerror=alert(document.location) src=x>"
+			).build(),
+			RandomTestUtil.randomLocaleStringMap(), true, new ServiceContext());
```

Comment:
Hey @thallescezar let's avoid this updating by persisting this data when adding, for example:

```java
		CalendarResource calendarResource =
			_calendarResourceService.addCalendarResource(
				TestPropsValues.getUserId(), TestPropsValues.getGroupId(),
				ClassNameLocalServiceUtil.getClassNameId(CalendarResource.class), 0,
				null, null, HashMapBuilder.put(
					LocaleUtil.getDefault(),
					"'\"></option><img onerror=alert(123) src=x>"
				).build(), RandomTestUtil.randomLocaleStringMap(), true, 
				new ServiceContext());
```